src/coreclr/inc/corinfo.h

```diff
@@ -2789,6 +2804,11 @@ class ICorStaticInfo
             bool                                isGCType
             ) = 0;
 
+    virtual void getThreadLocalStaticInfo_ReadyToRun(
```

Comment:
All this is native AOT specific. I think it is misleading to call it ReadyToRun.

Reply:
I have renamed the methods to have `NativeAOT` instead of `ReadyToRun`.